src/cljc/rems/common/util.cljc

```diff
@@ -435,4 +435,29 @@
   (is (= (assoc-some-in {} [:a] 1) {:a 1}))
   (is (= (assoc-some-in {} [:a :b] 1) {:a {:b 1}}))
   (is (= (assoc-some-in {:a {:b 1}} [:a :b] 2) {:a {:b 2}}))
-  (is (= (assoc-some-in {:a {:b 1}} [:a :b] nil) {:a {:b 1}})))
\ No newline at end of file
+  (is (= (assoc-some-in {:a {:b 1}} [:a :b] nil) {:a {:b 1}})))
+
+(defn replace-key
+  "Replaces key `k1` with key `k2` in map `m`.
+   If `k1` does not exist in `m`, returns `m`.
+   
+   **Examples:**
+   ```clojure
+   (replace-key {:a 1} :a :b)
+   ;;=> {:b 1}
+   
+   (replace-key {:a 1} :b :c)
+   ;;=> {:a 1}
```

Comment:
i saw in a blog post that VSCode Calva plugin understands Markdown syntax, and also core Clojure functions have same syntactical highlighting for code examples, so i thought that this could be a nice idea to try out. i don't know how Emacs behaves with MD though

![Screenshot 2021-12-01 at 14 47 53](https://user-images.githubusercontent.com/794087/144237085-32d3467b-16db-4a5b-b5cd-fa78d018270c.png)

Reply:
Emacs can work in a terminal so generally it does not do fancy displaying of things. CIDER doesn't seem to support this out of the box yet but at least it doesn't break anything.